### ISSUE TITLE
Error, the game updated the method and the patch expects a different number of parameters

### ISSUE THREAD

**Issue by ghost** [catgirlfighter/RimWorld_CommonSense#51]
https://pastebin.com/TQQd8wrj

**Comment by catgirlfighter** [comment]
I don't get that message.
Is your game 1.5?
Is your achtung updated for 1.5?
is your CS updated for 1.5?
Maybe issue of a mod that catches errors?

**Comment by ghost** [comment]
It's not about Achtung, the game updated the method and the patch expects a different number of parameters

**Comment by catgirlfighter** [comment]
As I said
>I don't get that message.

I also use achthung.

**Comment by ghost** [comment]
Then I don't know, I don't get it without Common Sense
https://pastebin.com/TQQd8wrj
https://pastebin.com/36f7kakg

**Comment by catgirlfighter** [comment]
Well, I didn't get a single answer to any of my questions. From log dump I assume you do use new version of the game (because anomaly is in the list), but can't really tell about CS.
Also I didn't check ANYTHING on DLC yet, so I have no clue if it's DLC thing.

**Comment by ghost** [comment]
Well, then it's a big chance that it's DLC, and everything updated as it can be

**Comment by catgirlfighter** [comment]
Yeah. I'll be buying DLC at some point soon, so until then these reports just kinda gonna be hanging in suspense. Kinda suspected there will be ones though. But hoped that just making it work in 1.5 would be enough.

**Comment by catgirlfighter** [comment]
Bought and installed DLC. It's still all fine.
Used both local and steam versions of mod.
![image](https://github.com/catgirlfighter/RimWorld_CommonSense/assets/13454532/3106c6fd-0a36-4411-a18f-71f174ac2c42)
